### PR TITLE
Feature/epoch 0.22.0

### PR DESCRIPTION
**Pull request by ricricucit**
Adds compatibility for Epoch 0.22.0

### CODE REVIEW

File: es/ae/aens.js [hunk 1]
```diff
@@ -82,13 +82,13 @@ function classify (s) {
...
+async function update (nameId, target, options = {}) {
   const opt = R.merge(this.Ae.defaults, options)
-
   const nameUpdateTx = await this.nameUpdateTx(R.merge(opt, {
-    nameHash,
-    account: await this.address(),
-    pointers: JSON.stringify(R.fromPairs([[classify(target), target]]))
+    nameId: nameId,
+    accountId: await this.address(),
+    // pointers: R.fromPairs([[classify(target), target]])
```
Comment (mpowaga) [inline es/ae/aens.js]: Looks like this can be removed.

Comment (ricricucit) [reply es/ae/aens.js]: mh. no, this will not make `updates names` test pass...

**Comment by ricricucit** [comment]
@mpowaga @nduchak  I have an issue on this PR: [this](https://ci.aepps.com/blue/organizations/jenkins/aepp-sdk-js/detail/PR-79/1/pipeline/) is the log explaining why this doesn't pass. 

I'm trying different solutions, but hints on this problem, if you have any...it would be of great help.

File: es/ae/aens.js (hunk 1 above)
Comment (ricricucit) [reply es/ae/aens.js]: ...or am I missing something?

Comment (mpowaga) [reply es/ae/aens.js]: I mean this comment on line 90.

**Comment by mpowaga** [comment]
You can remove `await` keyword which is redundant in this case.

File: es/ae/aens.js (hunk 1 above)
Comment (ricricucit) [reply es/ae/aens.js]: okok, [done](https://github.com/aeternity/aepp-sdk-js/pull/79/commits/71549f330cd1464eac24b16240736633cc443d82)

**Comment by ricricucit** [comment]
@mpowaga this does not help. 
I could see that myself, but test won't pass if I remove `await`.

**Comment by mpowaga** [comment]
Did you try to remove both `async` and `await` from that function?

**Comment by ricricucit** [comment]
yes.

**Comment by ricricucit** [comment]
removing both it's not a solution. https://eslint.org/docs/rules/no-return-await

**Comment by noandrea** [comment]
will close this PR due to CI issue, please start a new one

**Pull request closed without merging**